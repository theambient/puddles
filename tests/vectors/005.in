7 7
9 9 9 9 9 9 9
7 7 9 8 9 8 8
9 9 9 10 9 9 9
8 1 2 3 4 8 8
7 5 8 4 9 7 7
9 1 1 1 1 8 8
8 8 8 8 8 8 8
