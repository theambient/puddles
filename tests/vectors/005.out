0000000
000#000
0000000
0####00
0#0#000
0####00
0000000
